magento/magento2
# Fix image order when 10+ images on dropdown configurable products

### Description (*)


This PR fixes an images order, which was described mostly in this issue https://github.com/magento/magento2/issues/22249 and fixed in this commit https://github.com/magento/magento2/commit/700a1535cb6f58224e8e302ad1b4556d8a118e01

The same issue appears for products using the Dropdown attribute types (not swatches)

### Related Pull Requests

- https://github.com/magento/magento2/pull/22287

### Manual testing scenarios (*)

1. Install Magento with sample data
2. Update 2 attributes color and size to use Dropdown as "Catalog Input Type for Store Owner"
3. Upload more than 10 images to `WJ12-XS-Blue` product
4. On configurable product page that images should have correct order

### Questions or comments

Does it need new issue?

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#37285: Fix image order when 10+ images on dropdown configurable products
